### PR TITLE
s

### PR DESCRIPTION
**Pull request by DeadlyFirex**
**Short Description**:

**Steps to Reproduce**:
1.
2.
3.

**Browser Settings**:

**Additional Comments**:  

**Pre-report checks**:
- [ ] I've tried in incognito mode, or another device.
    - [ ] This issue still happens
    - [ ] This issue doesn't happen anymore
- [ ] I've checked the URL for any spelling mistakes
    - [ ] This issue still happens
    - [ ] This issue doesn't happen anymore
    - [ ] I've been following navigation links (please include the path in Additional Comments)
        - [ ] This issue still happens
        - [ ] This issue doesn't happen anymore
- [ ] I've checked all other open issues and pull requests, and none mention this.

### CODE REVIEW

**Comment by advaith1** [comment]
See what i posted in #contributors, it has a weird title and description, also what do you mean by "tag"

**Comment by ItsPugle** [comment]
I'll be looking into this later today. Thanks for your pull request.
In the meantime, I kindly request you sign the [Contributor License Agreement](https://cla-assistant.io/ItsPugle/DiscordApp?pullRequest=38).

File: docs/accounts.md [hunk 1]
```diff
@@ -18,8 +18,29 @@ After that, fill in the required information, and then BOOM! Your discord accoun
...
-First, let get one thing very clear, you **can't** delete your account it what way,
-It caused troubles and mix-ups, but you can't delete your account, if you want to "delete" your
-account, change your personal informatiom like your email/nickname and don't log in for a week,
-this will make your account inactive what counts like deleting it, when you log-in once again, it gets active again.
+If you're running the Canary client, you are able to delete your account by pressing the 'Delete Account' button in the User Settings menu. Otherwise, you'll need to [send an email to Discord Support](mailto:support@discordapp.com?subject=Account%20Deletion%20Request)
 
+![Delete Account Button](https://screenshots.should-be.legal/823c03.png)
+
+# Nicknames & Usernames
+Discord haves the ability to let you set nicknames for any server (Yes!) which shows your actual nickname over your real name, if you click on any member of your server, you will see his nickname (if setted up) and his username as shown here:
```
Comment (advaith1) [inline docs/accounts.md]: what emoji is the one around "Yes!" supposed to be, they just look like squares to me

File: docs/accounts.md [hunk 2]
```diff
@@ -18,8 +18,29 @@ After that, fill in the required information, and then BOOM! Your discord accoun
...
+  
+###### Server Nicknames
+And **BOOM**, my nickname is now Deadly on that server, use this command in any channel, don't get shocked if a random message from a clyde will appear (I almost got a heart attack, what a bot) that's normal, only you are able to see it and if you look very good you will see a small button named "Delete this message" in blue color, it's quite small so it took me two hours to find it, I guess your a lot better in finding buttons so good luck! So, you clicked that button? 
+  
+###### Tags
+Did the annoying clyde dissapear? Nice! So lets get back to the point, we have here in Discord, a super intelligent smart nice tag system named Tags, which is linked to each account, each account haves a temporary tag, If you change your **username** your tag will refresh too, so its not forever, tags are used to make a differance between more people using the same username, which means you could use any username anytime, anywhere. with no limits! You could use any username but it's important to keep your tag ready at all times, it can be found in the corner of the screen.
```
Comment (advaith1) [inline docs/accounts.md]: See what i said about "tags", also why did you call Clyde annoying 🤔

File: docs/accounts.md [hunk 3]
```diff
@@ -18,8 +18,29 @@ After that, fill in the required information, and then BOOM! Your discord accoun
...
+![Hey! That's me!](https://cdn.discordapp.com/attachments/328217116924837889/366842074605158400/image.png)
+As you see here, my nickname is shown in the black circle, that's the nickname inside the server, which will be only shown in that server, if I will join another server Discord will automatically show my username as my "nickname" there, so you have to set up a nickname for every server, next question is: How Deadly? Com'on and get to the point, well, my personal fastest way is doing /nick <Your nickname you want to setup> For example, My name is DeadlyFirex and I want to nick myself as "Deadly" then I will do this: /nick Deadly
+  
+###### Server Nicknames
+And **BOOM**, my nickname is now Deadly on that server, use this command in any channel, don't get shocked if a random message from a clyde will appear (I almost got a heart attack, what a bot) that's normal, only you are able to see it and if you look very good you will see a small button named "Delete this message" in blue color, it's quite small so it took me two hours to find it, I guess your a lot better in finding buttons so good luck! So, you clicked that button?
```
Comment (advaith1) [inline docs/accounts.md]: 1. the delete button is super easy to see

2. why would you get shocked about that

File: docs/accounts.md (hunk 1 above)
Comment (ItsPugle) [reply docs/accounts.md]: Please use GitHub Flavoured Markdown for emojis.

File: docs/accounts.md (hunk 2 above)
Comment (ItsPugle) [reply docs/accounts.md]: @advaith1 Keep reviews to reviewing not opinions like Clyde

Comment (advaith1) [reply docs/accounts.md]: oh sorry, i thought it was commenting

Comment (ItsPugle) [reply docs/accounts.md]: all g, but yeah. The thing about Clyde needs to be fixed, and

Comment (advaith1) [reply docs/accounts.md]: and... ?

**Pull request closed without merging**